modules/constructor/constructor-bottom/constructor-bottom.model.js

```diff
@@ -1,16 +1,20 @@
 const mongoose = require('mongoose');
 const Language = require('../../../models/Language').schema;
 const CurrencySet = require('../../../models/CurrencySet').schema;
-const ImageSet = require('../../common/ImageSet').schema;
 
 const constructorBottomSchema = new mongoose.Schema({
   name: [Language],
   material: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'Material',
   },
+  color: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Color',
+  },
   image: String,
   basePrice: [CurrencySet],
   available: Boolean,
+  default:Boolean,
```

Comment:
same

Reply:
Thit id default as a key in order you want use this elemend by defauld when counstructor is loaded for the first time